sdk/cognitiveservices/azure-cs-textanalytics/src/main/java/com/azure/cs/textanalytics/models/TextSentimentResult.java

```diff
@@ -13,24 +13,24 @@
  */
 @Fluent
 public final class TextSentimentResult extends DocumentResult {
-    private TextSentiment textSentiment;
-    private IterableStream<TextSentiment> items;
+    private TextSentiment documentSentiment;
+    private IterableStream<TextSentiment> textSentiments;
 
-    public TextSentiment getTextSentiment() {
-        return textSentiment;
+    public TextSentiment getDocumentSentiment() {
+        return documentSentiment;
     }
 
-    TextSentimentResult setTextSentiment(TextSentiment textSentiment) {
-        this.textSentiment = textSentiment;
-        return this;
+    public IterableStream<TextSentiment> getTextSentiments() {
```

Comment:
@mssfang have kept this as `getTextSentiments` rather that `getSentenceSentiments` as I think with the sentence we are introducing another keyword to the user. What do you think?

Reply:
If we keep `getTextSentiments`, we are assuming user know the the getTextSentiment is getting the sentence sentiments

Reply:
I think the sentence sentiment is already introduced by the service team. So i think it should be fine to change to SentenceSentiment. After it gives the user the intuitive sense the List of TextSentiment is a list of Sentence Sentiment.